### PR TITLE
Add `genezio create` command

### PR DESCRIPTION
**Pull request by costinsin**
## Type of change

-   [x] 🍕 New feature

## Description
Adds a new `genezio create` commands that lets the user create new genezio projects starting from a template.

## Checklist

-   [x] My code follows the contributor guidelines of this project;
-   [x] New and existing unit tests pass locally with my changes;

### CODE REVIEW

File: src/commands/create.ts [hunk 1]
```diff
@@ -0,0 +1,508 @@
...
+            "Project name must start with a letter and contain only letters, numbers and dashes",
+        );
+    }
+
+    const projectPath = path.join(process.cwd(), options.name);
+    if (nativeFs.existsSync(projectPath)) {
+        // If the project contains only a README.md/README or a .git folder, it's safe to continue
+        const files = nativeFs.readdirSync(projectPath);
+        for (const file of files) {
+            if (file !== "README.md" && file !== "README" && file !== ".git") {
```
Comment (radu1122) [inline src/commands/create.ts:49]: i would also add LICENCE and .gitignore here

File: src/commands/create.ts [hunk 2]
```diff
@@ -0,0 +1,508 @@
...
+import { YamlProjectConfiguration } from "../models/yamlProjectConfiguration.js";
+import { CloudProviderIdentifier } from "../models/cloudProviderIdentifier.js";
+import nativeFs from "fs";
+import { setLinkPathForProject } from "../utils/linkDatabase.js";
+import log from "loglevel";
+import { platform } from "os";
+import { GenezioCreateOptions } from "../models/commandOptions.js";
+import { debugLogger } from "../utils/logging.js";
+
+type ProjectInfo = {
```
[the remark this answers is not in the document]
Comment (costinsin) [reply src/commands/create.ts:18]: I think that we should keep the types that are used only inside one file in the same file. Types that are used in the entire app, should be placed in the models folder.

File: src/commands/create.ts (hunk 1 above)
Comment (costinsin) [reply src/commands/create.ts:49]: I agree

**Comment by costinsin** [comment]
### Merge activity

* **Jan 12, 6:22 AM**: @costinsin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/Genez-io/genezio/700).

**Pull request merged**